packages/ssx-sdk/src/types.ts

```diff
@@ -196,3 +197,11 @@ export interface Storage {
     /** Delete a value from storage. */
     delete(key: string): Promise<any>,
 }
+
+/** ENS data supported by SSX */
+export interface SSXEnsData {
+    /** ENS name/domain */
+    ensName?: string | null,
+    /** ENS avatar */
+    ensAvatarUrl?: string | null
+}
```

Comment:
I think the interface should be something like this. A use can provide this or we can default to resolving the domain and avatar, locally if they enable it
```suggestion
export interface SSXEnsData {
    resolve: {
        domain: boolean,
        avatar: boolean,
    },
    resolveOnServer: boolean
}
```

Reply:
I realized I made this comment on the wrong interface. I meant [this one](https://github.com/Juliano1612/ssx/blob/82098e008a782eaf22d4168c26a14c43199cd81d/packages/ssx-server/src/types.ts#L22)!

Reply:
Same here. This type is for describing the ENS data, not the configuration. But I think it would be better like this:

```
export interface SSXEnsData {
    /** ENS name/domain */
    domain?: string | null,
    /** ENS avatar */
    avatarUrl?: string | null
}
```

It's redundant to say `ens.ensAvatarUrl` or `ens.ensName`.